app/src/main/java/org/mozilla/fenix/home/recentbookmarks/view/RecentBookmarks.kt

```diff
@@ -161,7 +161,7 @@ private fun RecentBookmarkImage(bookmark: RecentBookmark) {
                     Box(
                         modifier = imageModifier.background(
                             color = when (isSystemInDarkTheme()) {
-                                true -> PhotonColors.DarkGrey30
+                                true -> PhotonColors.DarkGrey60
```

Comment:
Let's make a mental note to take this up with DS to consider a new color token to account for this use case.